includes/types/CMB2_Type_Associated_Objects.php

```diff
@@ -607,3 +481,479 @@ public static function maybe_callback( $query, $post_args ) {
 	}
 
 }
+
```

Comment:
Once the other things are cleaned up, i'll move these to their own files, and probably rename them. Just a heads up.